components/ArticleContentCard.jsx

```diff
@@ -1,7 +1,37 @@
 
+import { useEffect, useState } from "react";
+import { getArticleById, patchArticleVote } from "../src/api";
+import { useParams } from "react-router-dom";
+
 
 function ArticleContentCard({article}) {
     const dateOnlyRegex = /(\d+)[-](\d+)[-](\d{2})/g;
+    const {article_id} = useParams();
+    const [voteCount, setVoteCount] = useState(0)
+    const [err,setErr] =useState(null);
+    
+    useEffect(() => {
+        getArticleById(article_id)
+        .then(data => {
+            setVoteCount(data[0].votes)
+        })
+    },[])
```

Comment:
I presume you would already have this info in the article card that you could pass to this component? Possibly an unnecessary API request here